app/experimenter/nimbus-ui/src/lib/utils.ts

```diff
@@ -19,3 +19,10 @@ export const optionalBoolString = (
   }
   return String(value);
 };
+
+// function getKeyByValue(
```

Comment:
Oops, thought I needed this at one point but I don't, will remove tomorrow when I push again.